src/open_inwoner/accounts/views/cases.py

```diff
@@ -348,16 +329,42 @@ def get_context_data(self, **kwargs):
                 ),
                 "statuses": statuses,
                 "documents": documents,
-                "internal_upload_enabled": internal_upload_enabled,
-                "external_upload_enabled": external_upload_enabled,
-                "external_upload_url": external_upload_url,
                 "allowed_file_extensions": sorted(config.allowed_file_extensions),
+                "anchors": self.get_anchors(statuses, documents),
             }
-            context["anchors"] = self.get_anchors(statuses, documents)
```

Comment:
We're building a dictionary just above so this doesn't have to be it's own assignment.